server/controllers/inventory/inventory/core.js

```diff
@@ -153,6 +155,33 @@ function getItemsMetadataById(uuid) {
   return db.one(sql, [db.bid(uuid), uuid, 'inventory']);
 }
 
+
+function getItemsMetadataSearch(params){
+  if(params.group_uuid){
```

Comment:
I think filter parser will automatically do this conversion for you.  Could you try?